aya/src/programs/uprobe.rs

```diff
@@ -55,8 +55,52 @@ pub enum UProbeAttachLocation<'a> {
     /// The location of the target function in the target object file, offset by
     /// the given number of bytes.
     SymbolOffset(&'a str, u64),
-    /// The offset in the target object file, in bytes.
+    /// The **file** offset in the target object file, in bytes.
```

Comment:
Repeating the word `file` twice in the sentence is redundant. If you want to bold `**file**` later in the sentence, that's okay. Furthermore if you want to change the sentence to `The target object **file** offset, in bytes.` that's fine. Feel free to a following note if you feel like there's need for more clarity. Perhaps:

```
///
/// Note that this is not the virtual address of the instruction as it
/// would be loaded by the loader; it is the offset in the loaded object file.
```